model_deployment/README.md

```diff
@@ -19,15 +19,15 @@
  ```
     pip3 install virtualenv 
     virtualenv venv
-    C:\> venv\Scripts\activate.bat
+    venv\Scripts\activate.bat
  ```
  
  #### 4.3 Setup virtual environment in Windows in PowerShell
   - Run the following commands in PowerShell.exe
  ```
     pip3 install virtualenv 
     virtualenv venv
-    PS C:\> venv\Scripts\Activate.ps1
+    venv\Scripts\Activate.ps1
```

Comment:
By default, external script execution is disabled in Powershell. Add commands to set execution policy.
`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. For more details refer [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.1).